src/aibs_informatics_core/models/base/_pydantic_fields.py

```diff
@@ -0,0 +1,85 @@
+from __future__ import annotations
```

Comment:
Do we do any typing introspection? This import could break that code since it defers evaluation of type hints.

https://stackoverflow.com/a/66744445

> Since from __future__ import annotations avoids the evaluation of type hints, anything that depends on evaluated type hints must [explicitly evaluate type hints](https://docs.python.org/3/library/typing.html#typing.get_type_hints). This is usually not possible outside of top-level module or class scopes, unless locals are explicitly handled.